prepare_mkl.sh

```diff
@@ -68,7 +68,7 @@ echo $VERSION_LINE  # Return Version Line
 
 # MKL
 HOME_MKL=$1
-if [ ! -d "$HOME_MKL" ]; then
+if [ ! -z "$HOME_MKL" -a ! -d "$HOME_MKL" ]; then
    mkdir $HOME_MKL
```

Comment:
probably better to just do a mkdir -p in case more than one portion of the path doesn;t exist. In that case, you won;t need the second check '-d "$HOME_MKL" '